adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -151,9 +160,29 @@ private void validateAcquireTokenRequest(final AuthenticationRequest authenticat
                     ADALError.DEVELOPER_AUTHORITY_IS_NOT_VALID_URL);
         }
 
+        if (mTokenCacheAccessor != null) {
+            mTokenCacheAccessor.setAuthRequest(authenticationRequest);
```

Comment:
correct

---
In reply to: [76132074](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/703#discussion_r76132074) [](ancestors = 76132074)